ext/strings.go

```diff
@@ -445,6 +458,12 @@ func (lib *stringLib) CompileOptions() []cel.EnvOption {
 					s := str.(types.String)
 					return stringOrError(upperASCII(string(s)))
 				}))),
+		cel.Function("reverse",
```

Comment:
Please introduce a new 'version' to optionally enable this feature. By default max version is used, but for teams that want to control function rollout having a new version of the library they can opt into is important.

Reply:
New version introduced.